src/FormMigration/FormMetaDecorator.php

```diff
@@ -784,4 +784,53 @@ public function getFormFeaturedImage()
 
         return $featuredImage;
     }
+
+    /**
+     * @unreleased
+     */
+    public function isActiveCampaignEnabled(): bool
+    {
+        return true;
+    }
```

Comment:
@JoshuaHungDinh this doesn't look right lol we want to actually check if it's enabled